R/rdeephaven/R/table_handle_wrapper.R

```diff
@@ -95,10 +97,53 @@ TableHandle <- R6Class("TableHandle",
     to_data_frame = function() {
       arrow_tbl <- self$to_arrow_table()
       return(as.data.frame(as.data.frame(arrow_tbl))) # TODO: for some reason as.data.frame on arrow table returns a tibble, not a data frame
-    }
+    },
+    internal_table_handle = NULL
```

Comment:
It must be public if any function (including our own) needs to access the internal thing. Otherwise it can be private. This is an upside to your proposed refactoring of the table ops.